### PR TITLE
KelvinVR - New USB commands + Driver script

### PR DESCRIPTION
**Pull request by NaturalAutomatics**
Library USB commands are now added for LED control
Driver script allows to send commands to the Unit

### CODE REVIEW

File: MetaTempDriver/scripts/send_command.py
Comment (NaturalAutomatics) [inline MetaTempDriver/scripts/send_command.py:1]: Please create README explaining how Driver scripts works and suppose to be used

File: MetaTempUnit/Hardware/library/KelvinVR/src/KelvinVR.cpp [hunk 1]
```diff
@@ -187,3 +203,135 @@ void Kelvin::runThermalCycle(bool isCooling, int intensity, int durationSeconds)
...
+  {
+    int delay = 1000;
+    if (command.length() > 10)
+    {
+      delay = command.substring(10).toInt();
+    }
+    blinkFront(delay);
+    Serial.println("LED blinking");
+  }
+  else if (command == "LED FADE BLUE")
```
Comment (NaturalAutomatics) [inline MetaTempUnit/Hardware/library/KelvinVR/src/KelvinVR.cpp]: Update README list of commands with updates!

**Review by NaturalAutomatics — COMMENTED** [review]
List of USB commands should be updated and tested next

File: MetaTempDriver/scripts/send_command.py
Comment (AlexNoyanov) [reply MetaTempDriver/scripts/send_command.py:1]: README updated

File: MetaTempUnit/Hardware/library/KelvinVR/src/KelvinVR.cpp (hunk 1 above)
Comment (AlexNoyanov) [reply MetaTempUnit/Hardware/library/KelvinVR/src/KelvinVR.cpp]: Rewrote with switch-case

**Comment by AlexNoyanov** [comment]
All comments Done!

**Review by NaturalAutomatics — COMMENTED** [review]
Reviewed, added main parts for new Mk1 Alpha firmware

**Pull request merged**